web/app/components/comment/__control/_select-label/comment__control_select-label.scss

```diff
@@ -0,0 +1,9 @@
+.comment__control_select {
+  position: absolute;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  opacity: 0;
+  cursor: pointer;
```

Comment:
Please add width: 100% here. For some reason, select styled like this still has a width more than its container in Chrome. See https://github.com/umputun/remark/issues/80#issuecomment-403213881